### PR TITLE
Guidance

### PR DESCRIPTION
**Pull request by aranas**
this PR is to open up my work on the main guidance & documentation. Will soon invite for review and tag people in specific sections that I think need review specifically

### CODE REVIEW

**Comment by aranas** [comment]
> Hey just FYI: I've gotten part of the way through with a test of deployment via [`quarto`](https://quarto.org/). Can push that up as a separate branch to have a peak at if you like.

awesome! Yes I think a separate branch would be great. I guess this PR is really more about the guidance bit of the documentation rather than the actual code documentation. Will change the name to make that less confusing.

**Comment by aranas** [comment]
I need to remove the sphinx make files from this PR they are not meant to be part of this, only the guidance

File: README.md [hunk 1]
```diff
@@ -1,136 +1,116 @@
 # Welcome to the `clim-recal` repository! 
 
-## Background
+Welcome to clim-recal, a specialized resource designed to tackle systematic errors or biases in Regional Climate Models (RCMs). As researchers, policy-makers, and various stakeholders explore publicly available RCMs, they need to consider the challenge of biases that can affect the accurate representation of climate change signals. Clim-recal provides both a **broad review** of available bias correction methods as well as **practical tutorials** and **guidance** on how to easily apply those methods to various datasets.
```
Comment (gmingas) [inline README.md:3]:
```suggestion
Welcome to clim-recal, a specialized resource designed to tackle systematic errors or biases in Regional Climate Models (RCMs). As researchers, policy-makers, and various stakeholders explore publicly available RCMs, they need to consider the challenge of biases that can affect the accurate representation of climate change signals. 

Clim-recal provides both a **broad review** of available bias correction methods as well as **software**, **practical tutorials** and **guidance** that helps users apply these methods methods to various datasets.
```

File: README.md [hunk 2]
```diff
@@ -1,136 +1,116 @@
...
 
-Regional climate models (RCMs) contain systematic errors, or biases in their output [1]. Biases arise in RCMs for a number of reasons, such as the assumptions in the general circulation models (GCMs), and in the downscaling process from GCM to RCM [1,2].
+Clim-recal is an **Extensive guide to application of BC methods**: 
 
-Researchers, policy-makers and other stakeholders wishing to use publicly available RCMs need to consider a range of "bias correction” methods (sometimes referred to as "bias adjustment" or "recalibration"). Bias correction methods offer a means of adjusting the outputs of RCM in a manner that might better reflect future climate change signals whilst preserving the natural and internal variability of climate [2]. 
+- Accessible information about the [why and how of bias correction for climate data](#why-bias-correction)
+- Technical resource for application BC methods (see our full pipeline for bias-correction of the ground-breaking local-scale (2.2km)[Convection Permitting Model (CPM)](https://www.metoffice.gov.uk/pub/data/weather/uk/ukcp18/science-reports/UKCP-Convection-permitting-model-projections-report.pdf))
```
Comment (gmingas) [inline README.md:8]:
```suggestion
- Is a software library for for the application of BC methods (see our full pipeline for bias-correction of the ground-breaking local-scale (2.2km)[Convection Permitting Model (CPM)](https://www.metoffice.gov.uk/pub/data/weather/uk/ukcp18/science-reports/UKCP-Convection-permitting-model-projections-report.pdf)). Clim-recal brings together different software packages in Python and R that implement a variety of bias correction methods, making it easy to apply them to data and compare their outputs.
```

File: README.md [hunk 3]
```diff
@@ -1,136 +1,116 @@
...
+- In partnership with the MetOffice to ensure the propriety, quality, and usability of our work
```
Comment (gmingas) [inline README.md:9]:
```suggestion
- Was developed in partnership with the MetOffice to ensure the propriety, quality, and usability of our work
```

File: README.md [hunk 4]
```diff
@@ -1,136 +1,116 @@
...
+- Framework for open additions (in planning)
```
Comment (gmingas) [inline README.md:10]:
```suggestion
- Provides a framework for open additions of new software libraries/bias correction methods (in planning)
```

File: docs/pipeline_guidance.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+conda env create -f environment.yml
+```
+
+> **Warning**:
+> To reproduce our exact outputs, you will require GDAL version 3.4. Please be aware that this specific version of GDAL requires a different Python version than the one specified in our environment file. Therefore, we have not included it in the environment file and instead, for the reprojection step, you'll need to set up a new environment:
+> ```
+> conda create -n gdal_env python=3.10 gdal=3.4
+> ```
+
+In order to paralellize the reprojection step, we make use of the [GNU parallel shell tool](https://www.gnu.org/software/parallel/).
```
Comment (gmingas) [inline docs/pipeline_guidance.md:50]: Could you mention here that this needs to be installed and point to just the page that explains how? Or just include the installation command for the terminal directly?

File: docs/pipeline_guidance.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+> ```
+> conda activate gdal_env
+> ```
+
+To execute the reprojection in parallel fashion, run the `reproject_all.sh` script from your shell. As an input to the script replace `path_to_netcdf_files` with the path to the raw netCDF files.
+
+```bash
+cd bash
+sh reproject_all.sh path_to_netcdf_files
+```
```
Comment (gmingas) [inline docs/pipeline_guidance.md:112]: I think we also need to mention the `chmod` lines mentioned here: https://github.com/alan-turing-institute/clim-recal/issues/30 or find an equivalent solution.

**Review by gmingas — CHANGES_REQUESTED** [review]
This looks good, thanks for the massive amount of work Sophie!

I added some suggestins/comments. 

Also, before merging this will require a significant merge with main, as main has changed a lot since the branch was created (10 Aug). Some of the commits in main since 10 Aug modified the readme file in the top directory so these would have to be reconciled with what you did here (keeping all useful info).

**Comment by aranas** [comment]
> Also, before merging this will require a significant merge with main, as main has changed a lot since the branch was created (10 Aug). 
urrg good point, once we think this can be merged, I can go through the merge conflicts and consolidate.

**Comment by aranas** [comment]
This branch has been merged into branch `documentation-quarto`, so I will close the PR

**Pull request closed without merging**